### DISCUSSION TITLE
2023/11/19/new-blog-with-rspress/index

### DISCUSSION THREAD

**Discussion by giscus** [sumy7/sumy7.github.io#4] [Announcements]
# 2023/11/19/new-blog-with-rspress/index

SumyGG's Blog

https://sumygg.com/2023/11/19/new-blog-with-rspress/index.html

**Comment by zhifou** [comment]
借鉴一下，赞

**Reply by sumy7** [reply]
感谢支持，这是博客的源码地址，https://github.com/sumy7/SumyBlog-rspress
有什么建议也欢迎反馈。

**Comment by aojunhao123** [comment]
佬，我也在用rspress建站，有不会的问题可以来找您请教一下吗

**Reply by sumy7** [reply]
哈哈，可以的。我这两周也没时间折腾了，后续可能考虑自己迁移个主题。

**Comment by AIboy996** [comment]
rspress编译速度如何，我用mkdocs真是太慢了😭
> 另外，你的博客主题超好看诶
